app/views/isa_studies/_sample_types_form.html.erb

```diff
@@ -52,7 +52,7 @@
                                                                                   display_isa_tag: true } %>
 					<% end %>
 
-					<% unless sample_type.uploaded_template? || !sample_type.editing_constraints.allow_new_attribute? %>
```

Comment:
I think I would leave these checks in the code, but make the `allow_new_attribute?` method return `true`, just in case we want to change the behaviour in the future.